controllers/argocd/configmap.go

```diff
@@ -144,15 +144,60 @@ func getRBACScopes(cr *argoprojv1a1.ArgoCD) string {
 	return scopes
 }
 
-// getResourceCustomizations will return the resource customizations for the given ArgoCD.
+// getResourceCustomizations loads Resource Customizations from argocd-cm ConfigMap
 func getResourceCustomizations(cr *argoprojv1a1.ArgoCD) string {
 	rc := common.ArgoCDDefaultResourceCustomizations
 	if cr.Spec.ResourceCustomizations != "" {
 		rc = cr.Spec.ResourceCustomizations
 	}
+
+	// TODO: change this to log.warn once it's available with zap logger
+	log.Info("[WARNING] You are using an old format for ResourceCustomizations, please use the new format.")
 	return rc
 }
 
+// preferred way to add health customizations with `resource.customizations.health`
```

Comment:
I'd get rid of  "preferred way". Once the old way is removed, this comment becomes very puzzling as it suggests that there are other ways to do it.